westack/tests/westack_datasource_test.go

```diff
@@ -43,6 +43,19 @@ func Test_Datasource_Initialize_ConnectError(t *testing.T) {
 
 }
 
+func Test_DatasourceClose(t *testing.T) {
+
+	t.Parallel()
+
+	ds, err := app.FindDatasource("db_expected_to_be_closed")
+	assert.NoError(t, err)
+	assert.NotNil(t, ds)
+
+	err = ds.Close()
+	assert.NoError(t, err)
```

Comment:
I think to check `err` for nil is not enough for this situation, you must also check for `FindDataSource`, the result of this also must be `nil`, wdyt?

Reply:
@fredyk, it is good to check `app.FindDatasource()` as well. Let me add one test example to explain thorough :
`func (s *TestSuite) TestDeleteUser() {
	var dateNow = time.Now().UTC()

	ctx, storageWriter, storageReader := s.ctx, s.storageWriter, s.storageReader

	input := users.NedapUser{
		PK:        "mock_pk",
		Email:     "mock_email",
		Hash:      "mock_hash",
		UpdatedAt: dateNow,
	}

	err := storageWriter.PutUser(ctx, input)
	s.Nil(err)

	// We are deleting user with id "mock_id"
	err = storageWriter.DeleteUser(ctx, "mock_pk")
	s.Nil(err)

	// We have no any user after deleting
	m, err := storageReader.GetAll(ctx)
	s.Nil(err)
	s.Assert().Empty(m)
}`

in here we deleting user with `DeleteUser()` function and return `error`, for above test case it is not enough to check only `error` is nil or not, plus, we need to check deleted user is exist or not in. So, I suggested to test function itself.